spec/javascript/components/LocationTileSpec.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+
+import LocationTile from '../../../app/javascript/react/components/LocationTile';
+
+describe('LocationTile', () => {
+  let id, name, wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <BrowserRouter>
+        <LocationTile
+          id="1"
+          name="Chipotle"
+        />
+      </BrowserRouter>
+    );
+  });
+
+  it("Should render a component with the specific props for the location name", () => {
+    expect(wrapper.find(LocationTile)).toHaveProp('name')
+  });
+
+  it("Should render a component with the specific props for the location id", () => {
```

Comment:
This test isn't necessary for the reason above (and because you're not using the id)